### DISCUSSION TITLE
contentScrollOutsideEnable控制模式似乎没有生效？

### DISCUSSION THREAD

**Discussion by hpcat432** [DSAppTeam/PanelSwitchHelper#205] [Q&A]
构建PanelSwitchHelper时，contentScrollOutsideEnable置为false，希望使用容器固定模式。实际表现ContentContainer里的内容仍会被顶上去，滑出屏幕

**Comment by YummyLau** [comment]
容器固定模式性能相对较差，后续考虑移除了。出于什么考虑要降级为这个模式呢
